platform-dom/src/main/java/com/softicar/platform/dom/elements/DomLabel.java

```diff
@@ -2,18 +2,16 @@
 
 import com.softicar.platform.common.core.i18n.IDisplayString;
 import com.softicar.platform.dom.element.DomElementTag;
-import com.softicar.platform.dom.input.IDomInputNode;
+import com.softicar.platform.dom.input.IDomFocusable;
 import com.softicar.platform.dom.node.IDomNode;
 import com.softicar.platform.dom.node.tree.DomNodeTree;
 import com.softicar.platform.dom.parent.DomParentElement;
 
 /**
- * This class represents an Html label element.
+ * This class represents an HTML label element.
  * <p>
- * A label element can be connected to an {@link IDomInputNode}. If the user
- * clicks on the label, the connected input node will be selected. If the input
- * node is a text input, the input will be focused. If the input is a check-box
- * or radio-button, the <i>checked</i> state of the input will be toggled.
```

Comment:
The comment about checkboxes and radiobuttons was a lie. ;) I was only true for native checkboxes and radiobuttons, which we don't use anymore.